### ISSUE TITLE
4.2 Documentation Has Missing Items

### ISSUE THREAD

**Issue by AeroClassics** [emgucv/emgucv#326]
In the 4.2 documentation there are a large number of missing items. For example:
[Missing <summary> documentation for "T:Emgu.CV.Mat"]

This is true with virtually every page.

**Comment by emgucv** [comment]
Looking fine here:
http://www.emgu.com/wiki/files/4.2.0/document/html/2ec33afb-1d2b-cac1-ea60-0b4775e4574c.htm

**Comment by AeroClassics** [comment]
Yes, it looks great on line. But when you download the  install .exe I have a bunch of missing references in the documentation.

**Comment by emgucv** [comment]
This was reported against 4.2. The project is now on version 5 and the online documentation at http://www.emgu.com/wiki/files/4.2.0/document/html/ was confirmed working at the time. Closing now.
